<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GxsGroupDialog</class>
 <widget class="QDialog" name="GxsGroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/rstray3.png</normaloff>:/images/rstray3.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="HeaderFrame" name="headerFrame"/>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBoxLogo">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="horizontalSpacing">
            <number>6</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="0" rowspan="2">
            <widget class="QLabel" name="groupLogo">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="pixmap">
              <pixmap resource="../images.qrc">:/images/channels.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="groupName"/>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>9</number>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
             <item>
              <widget class="QPushButton" name="addLogoButton">
               <property name="text">
                <string>Add Icon</string>
               </property>
               <property name="icon">
                <iconset resource="../images.qrc">
                 <normaloff>:/images/add_image24.png</normaloff>:/images/add_image24.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="GxsIdChooser" name="idChooser"/>
             </item>
             <item>
              <widget class="QCheckBox" name="pubKeyShare_cb">
               <property name="toolTip">
                <string>Key recipients can publish to restricted-type Wiki Group, and can view and publish  for private-type channels</string>
               </property>
               <property name="text">
                <string>Share Publish Key</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3" rowspan="8">
       <widget class="QDockWidget" name="contactsdockWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>524287</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>check peers you would like to share private publish key with</string>
        </property>
        <property name="floating">
         <bool>false</bool>
        </property>
        <property name="features">
         <set>QDockWidget::NoDockWidgetFeatures</set>
        </property>
        <property name="windowTitle">
         <string>Share Key With</string>
        </property>
        <widget class="QWidget" name="dockWidgetContents">
         <layout class="QGridLayout" name="_2">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="FriendSelectionWidget" name="keyShareList" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>4</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>220</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="groupDescLabel">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="groupDesc"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QGroupBox" name="distribGroupBox">
        <property name="title">
         <string>Message Distribution</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QRadioButton" name="typePublic">
             <property name="text">
              <string>Public</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="typeGroup">
             <property name="text">
              <string>Restricted to Group</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="typeLocal">
             <property name="text">
              <string>Only For Your Friends</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="GxsCircleChooser" name="circleComboBox"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="localComboBox">
             <item>
              <property name="text">
               <string>Some Friends</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Another Group</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Family?</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Enemies</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QGroupBox" name="publishGroupBox">
        <property name="title">
         <string>Publish Signatures</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <widget class="QRadioButton" name="publish_open">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="publish_threads">
           <property name="text">
            <string>New Thread</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="publish_required">
           <property name="text">
            <string>Required</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="publish_encrypt">
           <property name="text">
            <string>Encrypted Msgs</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QGroupBox" name="personalGroupBox">
        <property name="title">
         <string>Personal Signatures</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <widget class="QRadioButton" name="personal_pgp">
           <property name="text">
            <string>PGP Required</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="personal_required">
           <property name="text">
            <string>Signature Required</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="personal_ifnopub">
           <property name="text">
            <string>If No Publish Signature</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QGroupBox" name="commentGroupBox">
        <property name="title">
         <string>Comments</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <widget class="QRadioButton" name="comments_allowed">
           <property name="text">
            <string>Allow Comments</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="comments_no">
           <property name="text">
            <string>No Comments</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QFrame" name="extraFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GxsIdChooser</class>
   <extends>QComboBox</extends>
   <header>gui/gxs/GxsIdChooser.h</header>
  </customwidget>
  <customwidget>
   <class>HeaderFrame</class>
   <extends>QFrame</extends>
   <header>gui/common/HeaderFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FriendSelectionWidget</class>
   <extends>QWidget</extends>
   <header>gui/common/FriendSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GxsCircleChooser</class>
   <extends>QComboBox</extends>
   <header>gui/gxs/GxsCircleChooser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
